### ISSUE TITLE
Audio accessibility

### ISSUE THREAD

**Issue by yayannabelle** [UKHomeOffice/posters#3]
First of all, I love these posters, and I hope many web designers and developers take them to heart. Thank you to those who helped make these posters a reality.

One thing that is missing from any of them is a mention against the use of autoplay, be it video or audio. As an autistic web user who has physical disabilities, there is little worse than opening an unsuspecting link and instantly having noise blasted at me, often times from a very small section of the page with even smaller controls, and having to find this source and stop it, already stressed out from the sudden noise. Often times I'll shut the tab without bothering further. (For clarification, I don't mean YouTube here. I mean, for example, opening a news article and suddenly a video is playing, when there was no indication of a video in the link I just clicked, and things like that.) 

This is a problem for autistic users because the sudden noise is likely to be upsetting (particularly for ones sensitive to sound, like myself), it's also a problem for those who use screenreaders as the two audios will overlap until they're able to turn the autoplaying sound off, and it's a problem for those with motor issues as you have to find the source of the noise, turn it off, and waste time and energy, if even able to as the controls are typically very small. In short, it's very irritating and should be avoided at all costs.

 [Here is a good article on the issue](https://www.abilitynet.org.uk/news-blogs/why-autoplay-accessibility-issue), that also goes into why autoplaying video is a bad idea as well, and [here's a page on the W3's take on autoplaying audio](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-dis-audio.html).

Perhaps an inclusion for a couple of the posters could be "do give the user a chance to choose to play the audio/video present on the web page / don't automatically have audio/video play upon  loading a page" (or something along those lines but more concisely worded!).

Thank you again for this work, please keep up the good work!

**Comment by karypun** [comment]
I'm glad you've enjoyed these posters Yayannabelle. We've been gathering feedback and suggestions for improving these posters and have duly noted your mention of autoplay :)
